### PR TITLE
docs: shorten readme; move vocabulary into docs; fix sextant page; update index

### PR DESCRIPTION
**Pull request by duncanjw**
Tidying up Chronicle first contact experience. Full revision of Chronicle docs to follow.

### CODE REVIEW

File: docs/chronicle_vocabulary.md [hunk 1]
```diff
@@ -1,104 +1,86 @@
 # Chronicle Vocabulary
 
-## Background
+Chronicle extends core PROV-O by adding vocabulary to describe cryptographic
+processes and identities, as well as prevent conflict between multiple Chronicle
```
Comment (suchapalaver) [inline docs/chronicle_vocabulary.md:4]:
```suggestion
processes and identities, as well as preventing conflict between multiple Chronicle
```

**Review by suchapalaver — CHANGES_REQUESTED** [review]

File: docs/chronicle_vocabulary.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+## chronicle:Identity
+
+Represents a cryptographic identity for an agent, supporting a single current
+signing identity via `chronicle:hasIdentity` and historical identities via
+`chronicle:hadIdentity`.
+
+## chronicle:publicKey
+
+A value containing a hex encoded ECDSA public key.
```
Comment (mtbc) [inline docs/chronicle_vocabulary.md:15]:
```suggestion
A value containing a hex-encoded ECDSA public key.
```

File: docs/chronicle_vocabulary.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+## chronicle:hadAttachment
+
+A historical attachment for a `prov:Entity`.
+
+Domain: `prov:Entity`
+Range: `chronicle:Attachment`
+
+## chronicle:entitySignature
+
+A hex encoded ECDSA signature for the resource represented by an attachment.
```
Comment (mtbc) [inline docs/chronicle_vocabulary.md:67]:
```suggestion
A hex-encoded ECDSA signature for the resource represented by an attachment.
```

File: README.md [hunk 4]
```diff
@@ -1,117 +1,47 @@
...
-Range: chronicle:Identity
-
-### chronicle:Namespace
+## Getting Started
 
-An IRI containing an external id and uuid part, used for disambiguation. In order
-to work on the same namespace discrete Chronicle instances must share the uuid
-part.
+We recommend you start by exploring the Chronicle examples repo and trying to
+build one of the examples there: <https://github.com/btpworks/chronicle-examples>
```
Comment (mtbc) [inline README.md:8]:
```suggestion
We recommend you start by exploring the
[Chronicle examples repo](https://github.com/btpworks/chronicle-examples)
and experimenting with one of the runnable application domains that it provides.
```
A few thoughts wrapped together in this suggestion, feel free to ignore them all, no blockers with your original.

Comment (duncanjw) [reply README.md:8]: I deliberately chose not to obfuscate the repo URL.

Comment (duncanjw) [reply README.md:8]: Feel free to change the text to something like this:
We recommend you start by exploring the Chronicle examples repo, and trying to
build and experiment with one of the runnable application domains there: <https://github.com/btpworks/chronicle-examples>

Comment (mtbc) [inline README.md:8]:
```suggestion
We recommend you start by exploring the Chronicle examples repo, and trying
to build and experiment with one of the runnable application domains there: <https://github.com/btpworks/chronicle-examples>
```

**Review by mtbc — APPROVED** [review]

**Review by suchapalaver — APPROVED** [review]

**Pull request merged**